src/app/models/propTypes/byline/index.js

```diff
@@ -0,0 +1,13 @@
+import { string, shape, arrayOf } from 'prop-types';
+
+const bylineBlockPropTypes = {
+  blocks: arrayOf(
+    shape({
+      title: string,
+      name: string,
```

Comment:
This implies both are optional, is that correct?

Reply:
Yes, the component will tolerate them both being optional, it rendering nothing if this is the case. We have a test case for this: https://github.com/bbc/simorgh/pull/5512/files#diff-fb0044ece4c63ce7e8c99ce4cb5cbfb5R46 

In theory Name and Title should always be provided by Ares according to https://github.com/bbc/simorgh/pull/5512#discussion_r378382359 but the component has been coded to tolerate this. Maybe our pre-processor rule should exclude the block if name or title are missing then we could safely make these mandatory? https://github.com/bbc/simorgh/blob/latest/src/app/lib/utilities/preprocessor/rules/cpsAssetPage/addBylineBlock/index.js#L13..L27

Reply:
Cool got it - agree that might be a better pattern for handling this type of situation. Could be a separate PR though.

Reply:
> Cool got it - agree that might be a better pattern for handling this type of situation. Could be a separate PR though.

@jamesdonoh  I'll raise an issue for this, I don't want to hold-up the delivery of this PR, particularly as it's already been manually tested.

Reply:
Issue to improve above: https://github.com/bbc/simorgh/issues/5582